vm.go

```diff
@@ -538,8 +543,13 @@ func (session *ioSession) SendTerminalSize(columns, rows int) error {
 
 // SendSignal
 func (session *ioSession) SendSignal(signal syscall.Signal) error {
+	containerID := session.containerID
```

Comment:
But we will never end up into the case you describe. In case someone wants to send a signal to an executed process on an existing container, this means this container has been created and that newcontainer has been called previously, meaning the session.containerID won't be empty.